recipes/genesis2/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "genesis2" %}
+{% set version = "2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/Genesis-1.3-Version2/archive/v{{ version }}.tar.gz
+  sha256: 0c8be1fd80cfa8339e3e40b5e798ec596d339e9b743c46a187321a71b9766e12
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
```

Comment:
Just for clarity, the only Fortran compiler we have on Windows currently is the `m2w64_fortran` one. It should work, but there may be some fiddling involved since it comes from MSYS2. Might be worth looking at other recipes that use this as well. FWIW there is also a `m2w64_c` compiler on Windows too

Reply:
@jakirkham the switch to `fortran` was a test based on a chat with @isuruf via Gitter to test `flang`.

Somehow for Windows it seems to be exploding the maximum size ```fatal error LNK1248: image size (9915A000) exceeds maximum allowable size (80000000)``` while with Unix all seems to be fine.

Reply:
Ah ok. Thanks for the context. Trying with `flang` also makes sense. It is a bit younger though.